ql/src/java/org/apache/hadoop/hive/ql/optimizer/physical/Vectorizer.java

```diff
@@ -4397,9 +4397,9 @@ public static Operator<? extends OperatorDesc> vectorizeFilterOperator(
         vContext, vectorFilterDesc);
   }
 
-  private static Operator<? extends OperatorDesc> vectorizeTopNKeyOperator(
-      Operator<? extends OperatorDesc> topNKeyOperator, VectorizationContext vContext,
-      VectorTopNKeyDesc vectorTopNKeyDesc) throws HiveException {
+  public static Operator<? extends OperatorDesc> vectorizeTopNKeyOperator(
```

Comment:
why is this change?

Reply:
To be able to test the class. I used the same pattern as `VectorFilterOperator` has. You can see the example in `TestVectorFilterOperator.getAVectorFilterOperator`.

Reply:
can we mark it as @VisibleForTesting